terraform/groups/ecs-service/locals.tf

```diff
@@ -9,7 +9,7 @@ locals {
   docker_repo                = "limited-partnerships-api"
   kms_alias                  = "alias/${var.aws_profile}/environment-services-kms"
   lb_listener_rule_priority  = 23
-  lb_listener_paths          = ["/transactions/*/limited-partnership/*", "/limited-partnership/healthcheck"]
+  lb_listener_paths          = ["/transactions/*/incorporation/limited-partnership/*", "/transactions/*/limited-partnership/*", "/limited-partnership/healthcheck"]
```

Comment:
will this be ok for the post incorporation? i thought it ended /limited-partnership and that rul looks like it would require /limited-partnership/

Reply:
Good spot. Updated.